### PR TITLE
Make seed not delete. Remove versions as the last step in seed

### PR DESCRIPTION
**Pull request by compwron**
### What github issue is this PR for, if any?
https://rubyforgood.slack.com/archives/CVB0QJGVD/p1610257613368700

### What changed, and why?
Make seed not delete. Remove versions as the last step in seed

### How will this affect user permissions?
no

### How is this tested? (please write tests!) 💖💪
run

### CODE REVIEW

**Review by littleforest — CHANGES_REQUESTED** [review]
This is not idempotent if we remove deleting the tables. I'm all for the change you are adding to not delete, but the populators need some love to rewrite `create!` statements to `find_or_create_by`.

File: db/seeds.rb [hunk 1]
```diff
@@ -23,9 +23,7 @@ def initialize
   end
 
   def seed
-    log "Erasing all objects from the database..."
-    destroy_all
-
+    log "NOTE: seed does not delete anything anymore! You have to run rake db:seed:replant to truncate and re-seed"
```
[the remark this answers is not in the document]
Comment (compwron) [reply db/seeds.rb:27]: Good point! I updated the readme and made seed not exactly idempotent but at least able to run multiple times without error- by checking existing objects

**Comment by compwron** [comment]
> This is not idempotent if we remove deleting the tables. I'm all for the change you are adding to not delete, but the populators need some love to rewrite `create!` statements to `find_or_create_by`.

Whups. Fixed, I think. :)

**Comment by keithrbennett** [comment]
I tried the following:

```
rake db:seed:replant > before && rake db:seed > after && diff before after
```
...and got the following:

```
13c13
<    76  CasaCase
---
>   150  CasaCase
20c20
<    76  CaseAssignment
---
>   152  CaseAssignment
23c23
<   374  CaseContact
---
>   744  CaseContact
```

So I don't think this script has remained idempotent.

**Pull request merged**